back/spec/support/mock_admin_publications_tree.rb

```diff
@@ -96,18 +96,21 @@ def create_other
 
   def create_published_parent_with_draft_children
     publication = build(:project_folder)
-    @published_parent_with_draft_children = build(:admin_publication,
+    parent = build(:admin_publication,
       publication_status: 'published',
       publication: publication)
-    publication.update!(admin_publication: @published_parent_with_draft_children)
-    children = Array.new(3) do
+    publication.update!(admin_publication: parent)
+    Array.new(3) do
       publication = build(:project)
       admin_publication = build(:admin_publication,
-        publication: publication,
-        publication_status: 'draft',
-        parent: published_parent_with_draft_children)
+        :with_parent, 
```

Comment:
Why do you need `with_parent`?

Reply:
I don't need it. Think I forgot to remove it 🤔